pkg/models/GPGKey.go

```diff
@@ -5,22 +5,16 @@ import (
 	"encoding/hex"
 	"fmt"
 	"github.com/quan-to/chevron/pkg/openpgp"
-	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
 	"strings"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).